raiden/ui/cli.py

```diff
@@ -424,27 +422,28 @@ def options(func):
                 "--flat-fee",
                 help=(
                     "Sets the flat fee required for every mediation in wei of the "
-                    "mediated token for a certain token network."
+                    "mediated token for a certain token address."
                 ),
                 type=(ADDRESS_TYPE, click.IntRange(min=0)),
                 multiple=True,
             ),
             option(
                 "--proportional-fee",
-                help="Mediation fee as ratio of mediated amount in parts-per-million (10^-6).",
-                default=DEFAULT_MEDIATION_PROPORTIONAL_FEE,
-                type=click.IntRange(min=0, max=10 ** 6),
-                show_default=True,
+                help=(
+                    "Mediation fee as ratio of mediated amount in parts-per-million "
+                    "(10^-6) for a certain token address."
```

Comment:
What does this mean? 

In settings, `# 0.4% in parts per million` is a comment on proportional fee. But what would the user get from the help above? how should the value provided by the user satisfy the `ratio` **in** `parts-per-million`? what is `parts-per-million` to begin with?

Reply:
The ppm thing is already in our other arguments for proportional fees. Not something I introduce. I have no idea why they decided to have it like this but I won't change it in this PR.

Perhaps ask @karlb or @palango why.

Reply:
We need an integer representation of a ratio. We deemed percent or promille as not suiting as they don't allow small enough values.

As for the description, I see your point but I'm not sure how to describe it better.